internal/distro/rhel90/distro.go

```diff
@@ -382,7 +400,9 @@ func (t *imageType) getPartitionTable(
 
 	imageSize := t.Size(options.Size)
 
-	return disk.NewPartitionTable(&basePartitionTable, mountpoints, imageSize, false, rng)
+	lvmify := !t.rpmOstree
```

Comment:
Hm, do we do LVM unconditionally for non-ostree? Or am I misunderstanding the logic?

Reply:
Ah, no, all good. `lvmify` only has an effect in case `mountpoints` is non-empty.